edgelet/edgelet-core/src/network.rs

```diff
@@ -0,0 +1,169 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+use crate::DEFAULT_NETWORKID;
+use serde_derive::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct Network {
+    name: String,
```

Comment:
Name should still be optional in this new format, so this should be `Option<String>`.

Reply:
Actually, maybe it shouldn't be optional. We're in this code path because the user wants to customize some parameters of the network. It makes sense to require them to be explicit about which network they want to be customized?

Reply:
Yeah, that's the reason I kept it as required.